packages/api/src/schemas.ts

```diff
@@ -2397,7 +2424,7 @@ export const methodSchemaDict: Record<string, MethodSchema> = {
             type: 'array',
             items: {
               type: 'object',
-              required: ['did', 'name', 'createdAt', 'indexedAt'],
+              required: ['did', 'name'],
```

Comment:
Not sure the cause of this change— I feel like it may have been leftover from #278.

Reply:
yup we get those values from the `profile` record which is not guaranteed to be there. so must be not required